### PR TITLE
Make elements list sorted

### PR DESCRIPTION
**Pull request by ViliusBa**
Sort selected items by their label in an ascending order.

### CODE REVIEW

File: packages/itwin/property-grid/src/components/ElementList.tsx [hunk 1]
```diff
@@ -58,22 +67,34 @@ export function ElementList({
...
+            {dataItem.label}
           </MenuItem>
         ))}
       </div>
     </div>
   );
 }
 
+/** Queries labels and orders Label-InstanceKey pairs in ascending order */
+async function getSortedLabelToInstanceKeyPairs(labelsProvider: PresentationLabelsProvider, instanceKeys: InstanceKey[]): Promise<RowElementData[]> {
```
Comment (saskliutas) [inline packages/itwin/property-grid/src/components/ElementList.tsx:87]:
```suggestion
async function getSortedLabelInstanceKeyPairs(labelsProvider: PresentationLabelsProvider, instanceKeys: InstanceKey[]): Promise<RowElementData[]> {
```

File: common/changes/@itwin/property-grid-react/make_elements_list_sorted_2023-08-23-13-59.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/property-grid-react",
+      "comment": "Sort selected items by their label in an ascending order.  ",
```
Comment (saskliutas) [inline common/changes/@itwin/property-grid-react/make_elements_list_sorted_2023-08-23-13-59.json:5]:
```suggestion
      "comment": "Sort selected elements list by label in an ascending order. ",
```

**Review by saskliutas — CHANGES_REQUESTED** [review]

File: common/changes/@itwin/property-grid-react/make_elements_list_sorted_2023-08-23-13-59.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/property-grid-react",
+      "comment": "Sort selected elements list by label in an ascending order. ",
```
Comment (saskliutas) [inline common/changes/@itwin/property-grid-react/make_elements_list_sorted_2023-08-23-13-59.json:5]: I missed one more space. 😞 

```suggestion
      "comment": "Sort selected elements list by label in an ascending order.",
```

**Review by saskliutas — APPROVED** [review]

File: packages/itwin/property-grid/src/components/ElementList.tsx [hunk 4]
```diff
@@ -84,15 +84,23 @@ export function ElementList({
...
-  return labelKeyPairs.sort((a, b) => { return ((a.label < b.label) ? -1 : ((a.label === b.label ? 0 : 1))); });
+  return labels
+  .map((label, index) => ({ label, instanceKey: instanceKeys[index] }))
+  .sort((a, b) => {
+    return a.label < b.label
+      ? -1
+      : a.label == b.label
+        ? 0
+        : 1;
+  });
```
[the remark this answers is not in the document]
Comment (ViliusBa) [reply packages/itwin/property-grid/src/components/ElementList.tsx]: Didn't know that such a method exists, thanks. Will adjust

[the remark this answers is not in the document]
Comment (ViliusBa) [reply packages/itwin/property-grid/src/components/ElementList.tsx]: Didn't know that such a method exists, thanks. Will adjust

**Pull request merged**